fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1415,6 +1423,10 @@ public void saveImage(File curFile, long replayedJournalId) throws IOException {
             dos.writeLong(checksum);
             checksum = shardManager.saveShardManager(dos, checksum);
             dos.writeLong(checksum);
+            if (Config.use_staros) {
```

Comment:
The content of the image should not depend on the config, because the config may change.

Reply:
fixed